### DISCUSSION TITLE
Should mol method `GetBondsMatchingQuery` be added?

### DISCUSSION THREAD

**Discussion by bertiewooster** [rdkit/rdkit#8133] [Ideas] [answered]
In analogy to [`GetAtomsMatchingQuery`](https://www.rdkit.org/docs/source/rdkit.Chem.rdchem.html#rdkit.Chem.rdchem.Mol.GetAtomsMatchingQuery), should there be a mol method `GetBondsMatchingQuery`? There are several bond queries in [rdkit.Chem.rdqueries](https://www.rdkit.org/docs/source/rdkit.Chem.rdqueries.html), such as [HasPropQueryBond](https://www.rdkit.org/docs/source/rdkit.Chem.rdqueries.html#rdkit.Chem.rdqueries.HasPropQueryBond), which could be used with such a method `GetBondsMatchingQuery`. Or is there an existing way to accomplish querying for bonds in a molecule that match certain criteria?

**Comment by greglandrum** [comment, marked as answer]
yeah, we should probably add this

**Reply by bertiewooster** [reply]
Thanks. I created issue #8168.
